tests/microos/networkmanager.pm

```diff
@@ -76,7 +85,7 @@ sub run {
     assert_script_run qq(nmcli device show | grep IP4.DNS | grep $dns_server);
     ping_check;
     # dnsmasq test
-    script_run(
+    script_output(
```

Comment:
Is this intentional, as you do not need the output for further processing

Reply:
as @dzedro mentioned in slack channel:
`Can you add one more change, replace script_run with script_output on line 79 88 ?
The commend is timing out on multiline commands because it is expecting the same output as input with special characters 'n etc.
script_output is waiting just for tag at the end of command
https://openqa.suse.de/tests/14165208#step/networkmanager/89` :)

Reply:
Yeah, true. The other way would be just to pull it from data folder as the file is static. But that is up to you as the implementer ofc. No objections